giturlparse/parser.py

```diff
@@ -82,8 +82,27 @@ def parse(self):
                   '([a-z0-9_.-]*)[:/]*'
                   '([\d]+){0,1}'
                   '([:/](.+)/(.+).git)')
+
         m2 = re.search(regexp, self._url)
 
+        regexp = (r'(git\+)?'
+                  '((?P<protocol>\w+)://)'
+                  '((?P<user>\w+)@)?'
+                  '((?P<site>[\w\.\-]+)\.com)'
+                  '(:(?P<port>\d+))?'
+                  '(\/(?P<owner>\w+))?'
+                  '(\/(?P<repo>[\w\-]+)(\.git)?)?')
```

Comment:
Why are the regexp captures getting named, but not used?

Reply:
Sorry, can you explain a bit more? I am not getting the point.